src/App.svelte

```diff
@@ -1,16 +1,21 @@
 <script lang="ts">
-  import { fly } from 'svelte/transition';
+  import { fly, fade } from 'svelte/transition';
   import type { WeatherResult } from "./lib/weatherAPI";
   import { getWeather } from "./lib/weatherAPI";
 
   let currentWeather: WeatherResult | null = null;
   let weatherVisible = false;
   const params = new URLSearchParams(window.location.search);
   let errorMessage = "";
+  let showMetric = false;
+  let showDuration = 20000;
```

Comment:
Please default to 10 * 1000 instead of 20 * 1000

Reply:
✅